tests/ui/proc-macro/derive-helper-legacy-spurious.rs

```diff
@@ -5,8 +5,8 @@
 #[macro_use]
 extern crate test_macros;
 
-#[derive(Empty)] //~ ERROR cannot determine resolution for the attribute macro `derive`
-#[empty_helper] //~ ERROR cannot find attribute `empty_helper` in this scope
```

Comment:
I removed these two expected errors because they were expected to be resolved to